td-shim/src/bin/td-shim/main.rs

```diff
@@ -95,7 +95,13 @@ pub extern "win64" fn _start(
     info: usize,
 ) -> ! {
     // The bootstrap code has setup the stack, but only the stack is available now...
-    let _ = td_logger::init();
+    let log_level = if cfg!(debug_assertions) {
```

Comment:
I think we need use `release_max_level_off`. Dont use runtime check.

Reply:
Thanks, I have updated the PR